### ISSUE TITLE
Develop Updated Getting Started page

### ISSUE THREAD

**Issue by daniellex0** [hackforla/website#1619]
### Overview
We need to update the Getting started steps because they are out of date, and update the layout to fit the new steps.

### Action Items

Design/Product

- [x] Review [Getting Started](https://www.hackforla.org/getting-started) page and determine what needs to be updated, and how 
   - [x] Review the user flow with the civic tech team
   - [x] Mockup quick ideas for how to update page
   - [x] Consider the devs' suggestions in their [document](https://docs.google.com/document/d/1asOea4JVGXy_0e3TnaNhekRwwJIxeDih93zvy9df1Ms/edit)
   - [x] Discuss and consult findings and ideas with design team and decide on direction together
- [x] Create final design of updated Getting Started page

Development

- [x] Develop the updated Getting Started page (see [new design in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13%3A0)- on the right in purple rectangle)
   - [x] Make sure to also update the content of the FAQ accordion on the bottom of the page (keep its layout the same, just remove the first section and update the text in the others where applicable)
- [x] Add new icons to [credits page](https://www.hackforla.org/Credits/)
- [x] Once PR #2059 is approved and merged, remove dependency from #2193

### Resources/Instructions
[Getting Started page](https://www.hackforla.org/getting-started)
New Getting Started page design in [Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13%3A0)- on the right in purple rectangle)
[Credit.yml file template](https://github.com/hackforla/website/wiki/credit.yml-file-template)
[Figma for Developers Guide](https://docs.google.com/document/d/1Is5Ah0tUH07kWYG_J6HTo0RF8Z9Foi2l/edit)

**Comment by Sihemgourou** [comment]
Needs to validate the user flow with the product team. @ashleylin1 lets schedule a meeting !

**Comment by Sihemgourou** [comment]
@ExperimentsInHonesty should we involve the civic opportunity team ?

**Comment by ashleylin1** [comment]
1. Progress: Created 3 drafts on Figma to shorten the content on the page; I lessened the number of icons and condensed the information but it's still pretty wordy with many tasks listed on the page
2. Blocks: need to discuss the desired user flow and whether parts of it can be automated (ex. being added to the roster, Github, Google Drive, etc)
3. Availability: anytime after 4pm PST this week
4. ETA: depends on how quickly we figure out the user flow

**Comment by Sihemgourou** [comment]
Suggestions (To address during next meeting) : 

Overview 
- [x] Why do we need the description text, what is its value ? "Hack for LA brings together civic-minded ..."
- [x] Adopt our Standard should be on Step 4

Step 1 : 
- [ ] Review the title to something like " Start your onboarding process", because you have to do the "before the meeting" before attending the onboarding session
- [x] Shorten text from this 
"Attend a Hack for LA Remote Volunteer Onboarding session via Zoom. Our next one can be found on our Meetup page."
to this
"Attend our Zoom Volunteer Onboarding. The next one can be found on our Meetup page."
- [ ] Shorten text from this 
"Read the hack for la code of conduct"
to this
"Read our code of conduct"
- [x] is there a link for the HFLA code of conduct ? Same question for the onboarding survey?
- [ ] There is little value to the description of the step as it is now. "After you attend onboarding..."

Step 2 : 
- [x] Put the (i) "If you do not have a GitHub account.." next to the "Receive access". Maybe try an overlay ? 
- [ ] Explain the CoP abbreviation before using it such as "Community of practice(CoP)"
- [x] Shorten text from this 
"3. Attend your CoP meetings weekly. See here for schedule. Network with your peers, build up your skills, and hear from industry professionals."
to this
"3. Network with your peers and attend your CoP meetings weekly (See here for schedule.)"

Step 3: 
- [x] Align the "Choose a Role" title above the reste of the text." 
- [ ] Review title it's too confusing with the step 4. Maybe replace it to something like "Shadow a project"

Step 4:
- [x] Align the "Join a Project" title above the reste of the text." 

Step 5: 
- [x] Align the title above the reste of the text." 
-  [x] Change step description

FAQ: 
Update the zoom tutorial

**Comment by daniellex0** [comment]
I'll address these since I attended the Tuesday meeting- but don't forget @Sihemgourou we all had decided we are planning to do UX Research for the next round of iterations to validate what really works best for volunteers, so we can make bigger changes once that happens. We don't want to assume too much and make too many big changes before we do some research and validate the changes.

>  Why do we need the description text, what is its value ? "Hack for LA brings together civic-minded ..."

This is how the page was originally created by a previous designer- I agree it doesn't make much sense to have the text that is there- I'd be curious to know if Bonnie has thoughts about what it should be replaced with.

>  Adopt our Standard should be on Step 3

Can you explain why? This was always the last step in the Getting Started page

> Step 1 :
>  Review the title to something like " Start your onboarding process", because you have to do the "before the meeting" before attending the onboarding session

I think it's fine to keep this "attend an onboarding session" because that is the most important call-to-action that they are doing in this step, and we want that to be really clear.

> Shorten text from this
> "Attend a Hack for LA Remote Volunteer Onboarding session via Zoom. Our next one can be found on our Meetup page."
> to this
> "Attend our Zoom Volunteer Onboarding. The next one can be found on our Meetup page."
>  Shorten text from this
> "Read the hack for la code of conduct"
> to this
> "Read our code of conduct"
>  

Sure if Bonnie agrees. I will say it can be good to remind about the Hack for LA name for new volunteers, so that they are very clear what the onboarding is for. But I agree that with the Code of Conduct we don't have to repeat 'Hack for LA'

> is there a link for the HFLA code of conduct ? Same question for the onboarding survey?

Yes there are links, like in the current Getting Started page

>  There is little value to the description of the step as it is now. "After you attend onboarding..."

I thought you were the one that came up with these captions on Tuesday? To make it clear there was a connection between each step? 

Step 2 :

> Put the (i) "If you do not have a GitHub account.." next to the "Receive access". Maybe try an overlay ?

Ok we can try to do this if Bonnie agrees, but I do think it utilizes the space well and moving it elsewhere could make it messy. Also they might not click the overlay. Maybe instead there can be just the (i) icon next to step2, but the actual (i) description stays where it is

>  Explain the CoP abbreviation before using it such as "Community of practice(CoP)"

Good idea, we'll add this

>  Shorten text from this
> "3. Attend your CoP meetings weekly. See here for schedule. Network with your peers, build up your skills, and hear from industry professionals."
> to this
> "3. Network with your peers and attend your CoP meetings weekly (See here for schedule.)"

This is up to Bonnie if she agrees to feature less information about the CoP

> Step 3:
>  Align the "Choose a Role" title above the reste of the text."

Can you explain more? Align next to the text on the right of the title?


>  Review title it's too confusing with the step 4. Maybe replace it to something like "Shadow a project"

I thought we all agreed to make it 'Choose a Role' on Tuesday.. I think it makes sense because you have to select a role from the role options available at the Community of Practice.

> Step 4:
>  Align the "Join a Project" title above the reste of the text."

Again not sure what this means, please explain a little more?

> Step 5:
>  Align the title above the reste of the text."

^same as above

>  Change step description

Please clarify what you mean by this?

> FAQ:
> Update the zoom tutorial

In what ways, just update the directions to make sure that it's consistent with how zoom works today? Or add something new?

**Comment by ashleylin1** [comment]
1. Progress: I created a new design that retains the original step order and includes step 2 of joining the community of practice. I also replaced the old icons with new ones that are from the same icon family to improve consistency (the old ones had different line widths) and the links to the icon source for attribution. One of the design versions also has a potential side navigation bar similar to the About page to make it easier for users to navigate between steps. I'm still working on the mobile version and I have not implemented the items discussed above but I will continue to work on them this week.
2. Blockers: I will discuss the items above with Danielle for clarification
3. Availability: I will likely not be able to attend Sunday meetings for a while due to a timezone difference but I am mostly available after 4pm PST

**Comment by ashleylin1** [comment]
Progress: Created a mobile version of the updated design and clarified w/Danielle about the above items
Blockers: need feedback on the design and whatever is decided in regards to UX research for this page
Availability: I will likely not be able to attend Sunday meetings for a while due to a timezone difference but I am mostly available after 4pm PST

**Comment by Sihemgourou** [comment]
Comment for @ExperimentsInHonesty 
<details>
<summary>Here is my suggestion for the top of the page</summary>

![Capture d’écran 2021-06-23 à 14 10 58](https://user-images.githubusercontent.com/55627107/123094494-3b64a300-d42d-11eb-9986-28b523e48435.png)

</details>

**Comment by daniellex0** [comment]
During our discussion at yesterday's meeting with @Sihemgourou  and @sayalikotkar  we all agreed to try to finish and send off the MVP version of the Getting Started page to developers as soon as possible so that the website can up up-to-date, and because it may take some time for the person assigned to work on reformatting the layout for this page. Any future changes to the onboarding process (the automation of Github/roster etc) can be implemented easily in the next iteration. Also FYI we won't be considering the sidenav/collapsible mobile section ideas until we do some UX Research on this page and those ideas in the near future.

@ashleylin1 please make these changes to the page, and please provide an update on here when you're finished:

Please make these changes to a duplicate of the blue rectangle version (where some text adjustments have already been made)- and please make the yellow text black again in the duplicate (since we are not adding the future changes yet until those are finalized)-

- [x] Incorporate this bolded line into the header (this version was approved, but feel free to move around as you see fit if you want)

<img width="350" alt="Screen Shot 2021-06-28 at 10 53 22 AM" src="https://user-images.githubusercontent.com/67400366/123681818-1d1cef80-d7ff-11eb-8463-1a94d16792d4.png">

- [x] Make the step illustrations in the header vertical instead of side-by-side (in mobile version)
   - [x] We can always hide these later if we want to in development if we decide to after UX research, but it's good to have this in the Figma version for now
- [x] Add the step illustrations to each step in the mobile version
- [x] Replace the current Communities of Practice icon (people around a table) with the one next to it that says "winner"

<img width="50" alt="Screen Shot 2021-06-28 at 10 50 49 AM" src="https://user-images.githubusercontent.com/67400366/123681573-d7602700-d7fe-11eb-80f7-f77d3f522f26.png">

- [x] Please raise the “Step 1” etc headers higher (above the text to their right), so it’s clear they are above/include everything below them

<img width="400" alt="Screen Shot 2021-06-23 at 7 35 21 PM" src="https://user-images.githubusercontent.com/67400366/123682590-062acd00-d800-11eb-99bc-5789fd3cef38.png">

- [x] Switch steps 4 & 5 in mobile (already switched in desktop)

- [x] Try to improve/layout the (i) section about Github in step 2 better if you can

- [x] We are removing the FAQ section about zoom

- [x] For the before the meeting/after the meeting question about Step 1 (because the survey is supposed to be filled out after the meeting, but the rest of the steps are for before)- maybe try adding a 'before the meeting' section on top for steps 1-2, and 'after the meeting' section below for step 3? We can see how that will look

Thank you!

**Comment by ashleylin1** [comment]
Progress: All changes can be found on the updated designs in the purple box on the Figma Getting Started Page
Blockers: None
Availability: after 4pm

https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13%3A0

**Comment by Sihemgourou** [comment]
Suggestion from product team: 
- Adopt our standards should be before choose a role
- The page seems good though. Let's make a ticket once the previous point is corrected.

**Comment by Sihemgourou** [comment]
@daniellex0 please put this issue in the prioritized backlog if no dev is assigned to it yet :)

**Comment by ashleylin1** [comment]
Progress: removed header steps in mobile design, added details to credit the new icons, switched "Adopt our standards" and "Choose A Role" steps order
Blockers: none

**Comment by Abel-Zambrano** [comment]
IMPORTANT!! 
This issue is working off of a feature branch!
Please follow this wiki before creating a new branch!
[how to work off a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)

**Comment by Sihemgourou** [comment]
Hey @ashleylin1, is the deisgn ready then?

**Comment by aalieu** [comment]
1. Actively working on creating the new 'Getting Started' page to match the designs. 
2. No blockers as of now – just actively trying CSS to get the layout correct.
3. Mostly full availability.
4. Aiming to be complete by next Sunday's meeting.

**Comment by aalieu** [comment]
1. Nearly complete. Just working on getting icons right with Ashley and some CSS corrections. 
2. No hard blockers. I know what I need to do to complete.
3. Mostly full availability.
4. Will submit PR before Tuesday's meeting.

**Comment by aalieu** [comment]
@Aveline-art  

Some questions/feedback to discuss during the meeting for this issue.

- The info icon presented in the designs had a watermark for the free version. I used a placeholder for now but it's a little bit blurry and doesn't look the greatest. Might want to replace it. 
- In the designs, the arrows to expand in the FAQ for the mobile version are red while the desktop version is black. Do we want to keep that one color for both or is this fine?
- For step 2, the 'Join your Community of Practice's Slack Channel' hyperlink leads to https://www.hackforla.org/communities-of-practice. There was no relevant link in the old 'Getting-Started' page for this so I just chose this which I thought was the most relevant. 
- There's a page called 'getting-started' which is the one I worked on, but there's also a page called 'getting-started-with-video' that is live but I don't think anything really redirects to it so I don't think it gets visited/used much. Some of the styles for the SCSS page are used solely in the 'getting-started-with-video' page so I was just curious if that page is still needed.

**Comment by Sihemgourou** [comment]
@aalieu We Converted the PR into draft until all your questions are answered


Answers from product team: 
- For step 2, the 'Join your Community of Practice's Slack Channel' hyperlink leads to https://www.hackforla.org/communities-of-practice. There was no relevant link in the old 'Getting-Started' page for this so I just chose this which I thought was the most relevant. => OK that's fine 
- The getting-started-with-video is not needed for now. Do not do anything for now, we will make a new issue if it's required.

@hackforla/website-design 

for the design team to make a decision on the other questions

**Comment by ExperimentsInHonesty** [comment]
@hackforla/website-design please review and put in progress after you have your notes

**Comment by Aveline-art** [comment]
@aalieu It would help the design team answer your questions to have photos that highlight the context behind your questions (like screencaps with arrows or photos; [one humorous example](https://github.com/hackforla/website/issues/1908#issuecomment-877908152)). Thank you!

**Comment by aalieu** [comment]
Uploading some pictures here so hoping that helps! 

Design question 1:

The info icon presented in the designs had a watermark for the free version. I used a placeholder for now but it's a little bit blurry and doesn't look the greatest. Requested icon assistance from Ashley but still waiting on a response.

![first comp](https://user-images.githubusercontent.com/28831668/129132844-26bbb2ee-0376-4cf3-93a3-fd77e2fac7ae.jpg)

Design question 2:

In the designs, the arrows to expand in the FAQ for the mobile version are red while the desktop version is black. Do we want to keep that one color for both or is this fine?

![comp2](https://user-images.githubusercontent.com/28831668/129132854-1556aaf3-4961-4dd0-a3bd-1315ddf37828.jpg)

**Comment by kristine-eudey** [comment]
@aalieu apologies for the delay on getting back to you regarding these design questions.

**Question 1:** Please use the high quality info icon shown in the [Misc. new images/icons ](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=12314%3A95528) figma frame. 

**Questions 2:** Please make the arrows in both the desktop and the mobile versions pink (the primary button color). I just checked the [style guide](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=4532%3A164) to find guidance on this and it identifies pink arrows be used in the case of accordion menus, which appears to be the functionality in both desktop and mobile versions of this menu. I've updated the Figma file to reflect this.

If these don't answer your questions please let me know and I'll try to provide more clarity. Thanks!

**Comment by aalieu** [comment]
@kristine-eudey 

No worries!

For the first question, that was the original icon that I tried using but the free version contains a watermark which is what led me to finding a placeholder. 

![image](https://user-images.githubusercontent.com/28831668/129245798-be23458f-2254-43a3-9908-3a264ed032d7.png)

 As for question 2, I'll update both arrows to pink.

Thank you for your response!

**Comment by kristine-eudey** [comment]
@aalieu thanks for clarifying, I misunderstood. You should just be able to export the version from the Figma frame I linked to that doesn't have the attribution attached, as long as we credit that's okay. 
<img width="1057" alt="Screen Shot 2021-08-12 at 2 57 53 PM" src="https://user-images.githubusercontent.com/16949503/129253450-badf4007-faa1-4e75-ac57-cc146e505ac0.png">
For the version below I exported it from Figma as .png at 512 pixels in width. You can play around with the export settings based on the size you need. 
![info](https://user-images.githubusercontent.com/16949503/129254151-99e60fd4-e755-4772-810d-cfe005d0491a.png)

**Comment by aalieu** [comment]
@kristine-eudey 

Awesome, thank you for your help!

**Comment by aalieu** [comment]
1. Resolved changes requested by reviewers. Just waiting for the final okay for it to be merged.
2. No blockers!
3. N/A
4. Completed unless there is another problem.

**Comment by ExperimentsInHonesty** [comment]
@aalieu The FAQ text still needs to be updated.  Here is screenshot from Figma (check the red box)

![image](https://user-images.githubusercontent.com/37763229/135784108-d9a9abc5-0d32-4fea-94f3-dab62933235a.png)

**Comment by ExperimentsInHonesty** [comment]
@aalieu Thank you for the changes!
